testsuite/ext-tools/maintenance_json_generator.py

```diff
@@ -55,6 +55,8 @@
                           "/SUSE_Updates_SUSE-MicroOS_5.2_x86_64/"],
     "slemicro53_minion": ["/SUSE_Updates_SLE-Manager-Tools-For-Micro_5_x86_64/",
                           "/SUSE_Updates_SUSE-MicroOS_5.3_x86_64/"],
+    "slemicro54_minion": ["/SUSE_Updates_SLE-Manager-Tools-For-Micro_5_x86_64/",
+                          "/SUSE_Updates_SUSE-MicroOS_5.4_x86_64/"],
```

Comment:
last element should not need the last comma here
```suggestion
                          "/SUSE_Updates_SUSE-MicroOS_5.4_x86_64/"]
```